packages/utils/CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # CHANGELOG
 
+## v0.x.x - _TBD, 2018_
+
+    * Fix a bug causing abi_decoder to throw on unknown events with the same signature (#366)
```

Comment:
`Fix a bug related to event signature collisions (argument indexes aren't included in event signatures) in the abi_decoder. The decoder used to throw on unknown events with identical signatures as a known event (except indexes).`